Sources/RayTracer.Library/Shapes/Triangle.cs

```diff
@@ -7,14 +7,17 @@ public class Triangle : IIntersectable
     public Vector3 A { get; }
 
     public Vector3 B { get; }
-    
+
     public Vector3 C { get; }
+    private Vector3 Normal { get; }
 
     public Triangle(Vector3 a, Vector3 b, Vector3 c)
     {
         A = a;
         B = b;
         C = c;
+
+        Normal = FindNormal();
```

Comment:
why have the Normal prop on Triangle? we may not ever use it in the end, please remove it